### PR TITLE
Replace "Live feeds" with the previous remote and federated TLs

### PR DESCRIPTION
**Pull request by omentic**
This PR:
- lints README.md (lol)
- fixes a critical aesthetic vulnerability where one line of text is slightly too long making stuff look ugly (if desired)
  - [previously](https://github.com/ralsei/types.pl/assets/35242550/cb7a41ba-19d7-4ce5-a096-d8d4ed950858), [fixed](https://github.com/ralsei/types.pl/assets/35242550/2d5a3255-b6d4-40b5-852b-5766bdcbb1a8)
- renames the clunky "Private mentions" back to "Direct messages" (if desired)
- replaces the new "Live feeds" and "Search" sidebar entries with the previous "Local" and "Federated" timeline entries

I have not tested this. Also do let me know if you don't want either of the fluff text changes.

### CODE REVIEW

**Comment by 132ikl** [comment]
my name is rose and i approve this effort

File: app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx [hunk 1]
```diff
@@ -19,7 +19,9 @@ const messages = defineMessages({
   notifications: { id: 'tabs_bar.notifications', defaultMessage: 'Notifications' },
   explore: { id: 'explore.title', defaultMessage: 'Explore' },
   firehose: { id: 'column.firehose', defaultMessage: 'Live feeds' },
-  direct: { id: 'navigation_bar.direct', defaultMessage: 'Private mentions' },
+  local: { id: 'navigation_bar.community_timeline', defaultMessage: 'Local' },
```
Comment (ionathanch) [inline app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx:23]: should be 'Local timeline'

File: app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx [hunk 2]
```diff
@@ -19,7 +19,9 @@ const messages = defineMessages({
...
+  federated: { id: 'navigation_bar.public_timeline', defaultMessage: 'Federated' },
```
Comment (ionathanch) [inline app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx:24]: should be 'Federated timeline"

**Comment by ionathanch** [comment]
Can we only change the en.json locale and not the others? That's also the only file I've changed for "post" -> "toot" and I don't want to have to change all the locale files again next time we update (esp. since some of those entries look not yet localized). I think the tests should still be happy as long as en.json matches the default messages...

**Comment by jyoo980** [comment]
jams

File: app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx (hunk 1 above)
Comment (omentic) [reply app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx:23]: i thought so too when writing the userscript but it was previously just "Local" and "Federated"

Comment (omentic) [reply app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx:23]: can change if you'd like ofc

**Comment by omentic** [comment]
pr has been de-bloated. in retrospect, blindly running `sed` and `git stage *`'ing the results is not a great development style

File: app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx (hunk 1 above)
Comment (ionathanch) [reply app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx:23]: oh surprising, weird that it's different from the value in `navigation_bar.community_timeline`

**Pull request merged**